lib/index.js

```diff
@@ -9,6 +9,20 @@ if (typeof Promise === "undefined") {
   require("es6-promise").polyfill();
 }
 
+if (typeof Buffer === "undefined") {
+  const g = global || window;
+  g.Buffer = require("buffer/").Buffer;
```

Comment:
Did you find the trailing "/" necessary?  It shouldn't be, and I've not run into problems elsewhere, but I haven't tried integrating with `react-native` either.

Reply:
I tested this a bit more, and it seems you are right. The trailing `/` is not necessary here.

I tested with node, react and react-native and it seems to work just fine. I think I can remove it here.